modules/c++/math/include/math/Constants.h

```diff
@@ -23,6 +23,31 @@
 #ifndef __MATH_CONSTANTS_H__
 #define __MATH_CONSTANTS_H__
 
+#include <sys/sys_config.h>
+
+#ifdef __CODA_CPP11
+#include <math.h>
+namespace math
+{
+struct Constants
```

Comment:
We don't want to define all the constants twice... can you figure out some way to avoid this?
Also, I don't think you need the word static with constexpr do you?